src/targets/diy/platformio.ini

```diff
@@ -0,0 +1,12 @@
+
+[platformio]
+workspace_dir = ../../.pio/
+include_dir = ../../include
+src_dir = ../../src
+lib_dir = ../../lib
+boards_dir = ../../boards
+data_dir = ../../
+extra_configs =
+	../common.ini
+	diy_900.ini
+	diy_2400.ini
```

Comment:
Delete file

Reply:
No! That is there for a reason...
This way you can open a specific targets and no need to handle long list of targets ;)
Same applies to other similar comments...